<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogEditIP</class>
 <widget class="QDialog" name="dialogEditIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jail Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/preferences-system.png</normaloff>:/preferences-system.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Jail Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabConfig">
         <attribute name="title">
          <string>Options</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkVNET">
            <property name="text">
             <string>Enable VNET / VIMAGE support</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>271</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>IPv4</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkIPv4">
            <property name="text">
             <string>IPv4 Address</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineIP">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkIPv4Bridge">
            <property name="text">
             <string>IPv4 Bridge Address (Requires VNET)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineIPBridge">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkIPv4Router">
            <property name="text">
             <string>IPv4 Default Router (Requires VNET)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="lineIPRouter">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>140</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>IPv6</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkIPv6">
            <property name="text">
             <string>IPv6 Address</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineIP6">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkIPv6Bridge">
            <property name="text">
             <string>IPv6 Bridge Address (Requires VNET)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineIP6Bridge">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkIPv6Router">
            <property name="text">
             <string>IPv6 Default Router (Requires VNET)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="lineIP6Router">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>140</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Aliases</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboIPType">
            <item>
             <property name="text">
              <string>IPv4 Aliases</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IPv4 Bridge Aliases</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IPv6 Aliases</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IPv6 Bridge Aliases</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="listIP"/>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushAdd">
              <property name="text">
               <string>&amp;Add</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/add.png</normaloff>:/add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushRemove">
              <property name="text">
               <string>&amp;Remove</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Permissions</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>The following permissions can be enabled or disabled for this jail. For more details hover over each item or refer to the jail manpage.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="listPerms"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/preferences-system.png</normaloff>:/preferences-system.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/dialog-cancel.png</normaloff>:/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="warden.qrc"/>
 </resources>
 <connections/>
</ui>
